File: workshop2/python/main.py
from pprint import pprint

from manazer import Manazer
from downloader import Downloader
from parser import SpatnyParser

manazer = Manazer(Downloader(), SpatnyParser())

seznam_zviratek = manazer.nacti_data()

pprint(manazer.nejlevnejsi_zviratko(seznam_zviratek))

#odkomentujte po implementaci
# pprint(manazer.najdi_zviratko_podle_nazvu(seznam_zviratek, 'Lemur kata'))

File: workshop2/php/main.php
<?php
require 'IParser.php';
require 'MujSpatnyParser.php';
require 'SeznamZviratek.php';
require 'Zviratko.php';
require 'IDownloader.php';
require 'FGDownloader.php';
require 'Manazer.php';


$downloader = new FGDownloader();
$parser = new MujSpatnyParser();

$manazer = new Manazer($downloader, $parser);
$zviratka = $manazer->nactiData();

var_dump($manazer->nejlevnejsiZviratko($zviratka));

//odkomentujte po implementaci
//var_dump($manazer->najdiZviratkoPodleNazvu($zviratka, 'Lemur kata'));